openstack/identity/v3/projects/requests.go

```diff
@@ -298,3 +298,30 @@ func DeleteTags(ctx context.Context, client *gophercloud.ServiceClient, projectI
 	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
 	return
 }
+
+// CheckTag checks if a a tag exists in a project.
+func CheckTag(ctx context.Context, client *gophercloud.ServiceClient, projectID string, tag string) (r CheckTagResult) {
+	resp, err := client.Get(ctx, getTagURL(client, projectID, tag), nil, &gophercloud.RequestOpts{
+		OkCodes: []int{204},
```

Comment:
I believe that 404 should be an OK code here. It's a legit response and it doesn't indicate a failure